cosmic-app-list/src/wayland_handler.rs

```diff
@@ -151,12 +162,23 @@ impl ToplevelInfoHandler for AppData {
         toplevel: &zcosmic_toplevel_handle_v1::ZcosmicToplevelHandleV1,
     ) {
         if let Some(info) = self.toplevel_info_state.info(toplevel) {
-            let _ = self
-                .tx
-                .unbounded_send(WaylandUpdate::Toplevel(ToplevelUpdate::Update(
-                    toplevel.clone(),
-                    info.clone(),
-                )));
+            if !info
+                .state
+                .contains(&zcosmic_toplevel_handle_v1::State::Minimized)
+            {
+                let _ = self
+                    .tx
+                    .unbounded_send(WaylandUpdate::Toplevel(ToplevelUpdate::Update(
+                        toplevel.clone(),
+                        info.clone(),
+                    )));
+            } else {
+                let _ = self
+                    .tx
+                    .unbounded_send(WaylandUpdate::Toplevel(ToplevelUpdate::Remove(
+                        toplevel.clone(),
+                    )));
+            }
```

Comment:
Same way, *if* we drop the upper case (see other comment), this also needs to be reverted.